db/migrate/20241022113543_add_referrer_to_widget_logs.rb

```diff
@@ -0,0 +1,5 @@
+class AddReferrerToWidgetLogs < ActiveRecord::Migration[7.0]
+  def change
+    add_column :widget_logs, :referrer, :string
```

Comment:
Make it `text` just to be safe - URLs can get pretty long 

(Ahoy also uses `text`: https://github.com/ElixirTeSS/TeSS/blob/186715cd51624ca53e678da735e0a664d4ab8319/db/schema.rb#L52)